ddtrace/tracer/textmap.go

```diff
@@ -180,6 +180,8 @@ func (i *withOptionsInjector) injectTextMapWithOptions(spanCtx ddtrace.SpanConte
 
 	traceID := spanID
 	samplingPriority := 0
+	env := ""
+	parentVersion := ""
```

Comment:
```suggestion
	var env, pversion string
```
I think this is a more idiomatic declaration and on par with the rest of the repo. Also, do you think we can get away with `pversion` instead here?

Reply:
Done 👍